containers/ecr-viewer/src/app/tests/services/evaluateFhirDataServices.test.ts

```diff
@@ -456,25 +504,59 @@ Home: 123-456-6909`,
       });
     });
 
-    it("should return Hospital Encounter Data and match snapshot", () => {
+    it("should return Hospital Encounter Data when present and match snapshot", () => {
+      // Patient bundle without Hospital Encounter Data
+      const bundle = JSON.parse(JSON.stringify(BundlePatientWithCovid)) as Bundle;
+
+      // Grab the composition section
+      const compositionEntry = bundle!.entry!.find(
+          (e: any) => e.resource?.resourceType === "Composition"
+      );
+
+      // Add Hospital encounter data to section array. ts-ignore because it thinks section doesn't exist
+      // @ts-ignore
+      compositionEntry?.resource?.section?.push(admissionDiagnosis, dischargeDiagnosis)
```

Comment:
can we import bundle once and do something like this instead

```
const thisBundle = {
   ...bundle,
   entry: [
      { 
         ...bundle.entry[0],
         section: [
            ...bundle.entry[0].section,
            dxSection
         ]
      },
      ...bundle.entry.slice(1)
   ]
}
```

it's a bit clunky, but is non mutating and could probably be abstracted out to a function so you could reuse for the various tests